src/metabase/search/postgres/index.clj

```diff
@@ -111,14 +112,19 @@
       (update :display_data json/generate-string)
       (update :legacy_input json/generate-string)
       (assoc
+       :updated_at       :%now
        :model_id         (:id entity)
        :model_created_at (:created_at entity)
        :model_updated_at (:updated_at entity)
-       :search_vector    [:to_tsvector
-                          [:inline tsv-language]
-                          [:cast
-                           (:searchable_text entity)
-                           :text]])))
+       :search_vector [:||
+                       [:setweight [:to_tsvector [:inline tsv-language] [:cast (:name entity) :text]]
+                        [:inline "A"]]
+                       [:setweight [:to_tsvector
+                                    [:inline tsv-language]
+                                    [:cast
+                                     (:searchable_text entity)
+                                     :text]]
+                        [:inline "B"]]])))
```

Comment:
Comment here with a brief explanation for newbies to ABCD ranking etc would be nice. At least a link to the relevant documentation page.

Reply:
Also I'm thinking instead of just literally using `:name` here as more-ranked field I'd rather have `primary_text` and `secondary_text` (or maybe a bit more laconic, but also less-concrete), wdyt?